tests/lib/hardware/openstack_libcloud.py

```diff
@@ -41,6 +41,8 @@
 
 if config.DISTRO == 'openSUSE_k8s':
     from tests.lib.distro.opensuse import openSUSE_k8s as Distro
+elif config.DISTRO == 'SLES_CaaSP':
+    from tests.lib.distro.sles import SLES_CaaSP as Distro  # type: ignore
```

Comment:
why the `ignore` ? What's the complain that `mypy` has?

Reply:
https://stackoverflow.com/questions/59037244/mypy-incompatible-import-error-for-conditional-imports

I'll follow this up with a simple factory fix.